docs/upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -79,4 +79,10 @@ There you can find links to upgrade notes for other versions too.
         +       arguments:
         +           $useInlineEditation: false
         ```
+- use `CronModuleExecutor` as a service in DIC ([#1314](https://github.com/shopsys/shopsys/pull/1314))
+    - update your code if you are using `CronFacade::TIMEOUT_SECONDS` constant because it will be removed in next major release
```

Comment:
It could be made consistent with the latter format of "X was deprecated, do Y" and it's missing what should be done.

```suggestion
    - the constant `CronFacade::TIMEOUT_SECONDS` was deprecated and the timeout was moved to DIC configuration - if you need to access the timeout from different places, define your own constant and [use it in the YAML config via `!php/const`](https://symfony.com/blog/new-in-symfony-3-2-php-constants-in-yaml-files)
```